### PR TITLE
Docs 10746 example form data v2.4 cv

### PR DESCRIPTION
**Pull request by Cristian-Venticinque**

### CODE REVIEW

File: modules/ROOT/pages/dw-multipart-functions-form.adoc [hunk 1]
```diff
@@ -43,6 +103,8 @@ var secondPart = "content for my second part"
 }
 ----
 
+In this structure, the part keys `part1` and `part2` are stripped out in the `multipart/form-data` output.
+
```
Comment (mlischetti) [inline modules/ROOT/pages/dw-multipart-functions-form.adoc:107]: DW Source section grammar doesn't allow free text so this description should be Example section or in the Output section to be regenerated from DW's docs.

**Review by mlischetti — APPROVED** [review]

**Review by bjoson-mulesoft — APPROVED** [review]
Looks good overall. The long sentences with text offset in parenthesis are a bit hard to read, and the references to information in the example code are a bit hard to follow. Consider rewriting with bullet lists and/or using callouts in the code examples as documented here: https://docs.asciidoctor.org/asciidoc/latest/verbatim/callouts/
